server/sonar-server/src/main/java/org/sonar/server/rule/ws/RulesWsParameters.java

```diff
@@ -0,0 +1,102 @@
+  public static final String PARAM_REPOSITORIES = "repositories";
+  public static final String PARAM_RULE_KEY = "rule_key";
+  public static final String PARAM_ACTIVATION = "activation";
+  public static final String PARAM_QPROFILE = "qprofile";
+  public static final String PARAM_SEVERITIES = "severities";
+  public static final String PARAM_AVAILABLE_SINCE = "available_since";
+  public static final String PARAM_STATUSES = "statuses";
+  public static final String PARAM_LANGUAGES = "languages";
+  public static final String PARAM_TAGS = "tags";
+  public static final String PARAM_TYPES = "types";
+  public static final String PARAM_INHERITANCE = "inheritance";
+  public static final String PARAM_ACTIVE_SEVERITIES = "active_severities";
+  public static final String PARAM_IS_TEMPLATE = "is_template";
+  public static final String PARAM_TEMPLATE_KEY = "template_key";
+  public static final String PARAM_ORGANIZATION = "organization";
+  public static final String PARAM_COMPARE_TO_PROFILE = "compareToProfile";
+
+  public static final String FIELD_REPO = "repo";
+  public static final String FIELD_NAME = "name";
+  public static final String FIELD_CREATED_AT = "createdAt";
+  public static final String FIELD_SEVERITY = "severity";
+  public static final String FIELD_STATUS = "status";
+  public static final String FIELD_INTERNAL_KEY = "internalKey";
+  public static final String FIELD_IS_TEMPLATE = "isTemplate";
+  public static final String FIELD_TEMPLATE_KEY = "templateKey";
+  public static final String FIELD_TAGS = "tags";
+  public static final String FIELD_SYSTEM_TAGS = "sysTags";
+  public static final String FIELD_LANGUAGE = "lang";
+  public static final String FIELD_LANGUAGE_NAME = "langName";
+  public static final String FIELD_HTML_DESCRIPTION = "htmlDesc";
+  public static final String FIELD_MARKDOWN_DESCRIPTION = "mdDesc";
+  public static final String FIELD_NOTE_LOGIN = "noteLogin";
+  public static final String FIELD_MARKDOWN_NOTE = "mdNote";
+  public static final String FIELD_HTML_NOTE = "htmlNote";
+
+  /**
+   * @deprecated since 5.5, replaced by {@link #FIELD_DEFAULT_REM_FUNCTION}
+   */
+  @Deprecated
+  public static final String FIELD_DEFAULT_DEBT_REM_FUNCTION = "defaultDebtRemFn";
+  public static final String FIELD_DEFAULT_REM_FUNCTION = "defaultRemFn";
+
+  /**
+   * @deprecated since 5.5, replaced by {@link #FIELD_REM_FUNCTION}
+   */
+  @Deprecated
+  public static final String FIELD_DEBT_REM_FUNCTION = "debtRemFn";
+  public static final String FIELD_REM_FUNCTION = "remFn";
+
+  /**
+   * @deprecated since 5.5, replaced by {@link #FIELD_GAP_DESCRIPTION}
+   */
+  @Deprecated
+  public static final String FIELD_EFFORT_TO_FIX_DESCRIPTION = "effortToFixDescription";
+  public static final String FIELD_GAP_DESCRIPTION = "gapDescription";
+
+  /**
+   * @deprecated since 5.5, replaced by {@link #FIELD_REM_FUNCTION_OVERLOADED}
+   */
+  @Deprecated
+  public static final String FIELD_DEBT_OVERLOADED = "debtOverloaded";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)